release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -2720,6 +2720,78 @@ This section will continue to be updated over time to provide notes on enhanceme
+
+[id="ocp-4-15-12-operatorhub-filter-rename-FIPS"]
+===== OperatorHub filter renamed from "FIPS Mode" to "Designed for FIPS"
+
+* Previously, OperatorHub included a filter named "FIPS Mode". With this release, that filter is named "Designed for FIPS". (link:https://issues.redhat.com/browse/OCPBUGS-32933[*OCPBUGS-32933*])
+
+[id="ocp-4-15-12-bug-fixes"]
+==== Bug fixes
+
+* Previously, containers had an incorrect view of the pids limit in their `cgroup` hierarchy and reported as a random number instead of `max`. The containers do not have max pids and are limited by the pod pid limit, which is set outside of the container's `cgroup` hierarchy and not visible from within the container. With this release, the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-28926[*OCPBUGS-28926*])
+
+* Previously, for {product-title} deployments on {rh-openstack-first}, the `MachineSet` object did not correctly apply the value for the `Port Security` parameter. With this release, the `MachineSet` object applies the `port_security_enabled` flag as expected. (link:https://issues.redhat.com/browse/OCPBUGS-30857[*OCPBUGS-30857*])
+
+* Previously, the Installer erroneously attempted to verify the libvirt network interfaces when an agent-based installation was configured with the openshift-baremetal-install binary. With this release, the agent installation method does not require libvirt and this validation is disabled. (link:https://issues.redhat.com/browse/OCPBUGS-30944[*OCPBUGS-30944*])
+
+* Previously, the `cpuset-configure.sh` script could run before all of the system processes were created. With this release, the script is only triggered to run when CRI-O is initialized and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-31692[*OCPBUGS-31692*])
+
+* Previously, an incorrect `dnsPolicy` was used for the konnectivity-agent daemonset in the data plane. As a result, when CoreDNS was down, konnectivity-agent pods on the data plane could not resolve the proxy-server-address and could fail the konnectivity-server in the control plane. With this release, konnectivity-agent uses the host system DNS service to lookup the proxy-server-address and not depend on CoreDNS. (link:https://issues.redhat.com/browse/OCPBUGS-31826[*OCPBUGS-31826*])
+
+* Previously, if gathering logs from the bootstrap node failed during the `gather bootstrap` execution, the VM serial console logs were not included in the gather output even if they were collected. With this release, serial logs are always included if they are collected. (link:https://issues.redhat.com/browse/OCPBUGS-32264[*OCPBUGS-32264*])
+
+* Previously, port 22 was missing from the worker node's security group in SDK install, therefore connecting to the compute nodes with SSH failed when users used AWS SDK provisioning. With this release, port 22 is added to the worker security group and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-32383[*OCPBUGS-32383*])
```

Comment:
```suggestion
* Previously, port 22 was missing from the worker node's security group in SDK installations, therefore connecting to the compute nodes with SSH failed when users used AWS SDK provisioning. With this release, port 22 is added to the worker security group and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-32383[*OCPBUGS-32383*])
```
Should that first sentence also be "AWS SDK" instead of just "SDK"? Not familiar with the area.

Reply:
good call - I believe so